applications/plugins/SofaCUDA/sofa/gpu/cuda/CudaTriangleModel.h

```diff
@@ -22,18 +22,30 @@
 #pragma once
 
 #include <sofa/gpu/cuda/CudaTypes.h>
-#include <SofaMeshCollision/TriangleModel.h>
-#include <SofaMeshCollision/TriangleModel.inl>
+#include <sofa/component/collision/geometry/TriangleModel.h>
```

Comment:
The inl is not required here?

Reply:
not sure, we only add alias in this header

Reply:
I have the impression that, in general for template specialization, there is the following inclusions:
- specialized_template.h includes base_template.h, 
- specialized_template.inl includes base_template.inl, 
- specialized_template.cpp includes the specialized_template.inl